src/main/java/org/polyjdbc/core/dialect/DefaultDialectTypes.java

```diff
@@ -71,4 +71,9 @@ public String bool() {
         return "BOOLEAN";
     }
 
+    @Override
+    public String binary(Object value) {
```

Comment:
can you tell me why you use `Object` here? from what i see you usually have to specify the max length of BINARY column, so maybe it should be `binary(int maxLength)` like in `string(int characters)`?

Reply:
In Postgres it would be simply `bytea`, discarding `maxLength`, since in Postgres it is variable in length by default (more like `VARBINARY` from MySQL/Oracle)

Reply:
@adamdubiel 
![](https://img.kacperduras.pl/f8cc88f48e1d146768b049027ba45a25.png)

I did this because of DB2.